features/configure.feature

```diff
@@ -0,0 +1,148 @@
+Feature: configure command
+
+  Scenario: 'exo configure' with not enough arguments
+    When starting "exo configure" in the terminal
+    Then I see:
+      """
+      Error: Wrong number of arguments
+      Usage:
+        exo configure [remote-id] [flags]
+        exo configure [command]
```

Comment:
What's the command that's referred to here? Is it just `exo configure help`?

Reply:
See #883 for a related thought

Reply:
I'm confused. I thought `exo configure read qa` is a valid call. None of the usage examples match that?